src/defrag.c

```diff
@@ -68,6 +64,7 @@ void* activeDefragAlloc(void *ptr) {
     newptr = zmalloc_no_tcache(size);
     memcpy(newptr, ptr, size);
     zfree_no_tcache(ptr);
+    server.stat_active_defrag_hits++;
```

Comment:
`stat_active_defrag_hits`  may not be accurate.

Reply:
why?

p.s. note that the main purpose of that field is not just for statistics, but also for defrag loop itself (having a sense how how much work was done, and when it's time to call `mstime` and check the exit conditions).